docs/student.md

```diff
@@ -86,6 +86,46 @@ The organizing committee does not recommend that students start research and dev
 
   - According to the final project evaluation results, the remaining project bonus will be paid in proportion.
 
+## Student bonus instruction
+
+1. How much bonus amount will each student receive?
+
+Each project will be awarded RMB 12000, RMB 9000 and RMB 6000 (before tax), which based on the level of project difficulty.
+
+2. When will the bonus be paid?
+
+Students who pass the mid-term assessment will receive 50% of the project bonus first. According to the final project evaluation results, the remaining project bonus will be paid in proportion. If the student fails the final evaluation, the rest bonus will not be paid.
+
+3. How to pay the students’ bonus?
+
+The students’ bonus will be paid by credit transfer method. There is no restriction on which bank but the bank account must be under your own name. Also, the organizing committee will collect the following information from the mentor:
+
+- Name
+
+- ID number
+
+- Bank account and branch information
+
+*Note: the organizing committee will take reasonable measures to protect private data, personal identity information and other sensitive information
+
+4. How to pay the foreign students’ bonus?
+
+- The foreign students’ bonus shall be settled in dollars, subject to the real-time exchange rate at the time of bonus payment.
+- The students’ bonus will be paid by credit transfer method. There is no restriction on which bank but the bank account must be under your own name. Also, the organizing committee will collect the following information from the mentor:
+  - Name and address of receiving bank
```

Comment:
I think it will be better that this item is spited to two items.

Reply:
![image](https://user-images.githubusercontent.com/75349684/109774944-d4cc3700-7c3b-11eb-8b0a-08376db9d2ae.png)

According to the document provided by bank, I think the name and address of foreign bank may the coherent item.

Reply:
@YifeiXia I think the better English word of `收款人开户行名称和地址` are Payee’s Bank Name and Payee’s Bank Address, as ref [link](https://pic.bankofchina.com/bocappd/america/201307/P020130716342963797665.pdf). Although, we need to provide coherent phrase `Payee’s Bank Name + blank + Payee’s Bank Address` to bank when we transfer bonus to student or mentor, we also can tell student and mentor to provide these two item seperately and it will  be clearer in this way.